### PR TITLE
Task/csi 4746 add more informative logs

### PR DESCRIPTION
**Pull request by matancarmeli7**

### CODE REVIEW

File: controllers/array_action/array_mediator_svc.py [hunk 1]
```diff
@@ -1791,6 +1796,7 @@ def _rmhost(self, host_name):
             raise ex
 
     def delete_host(self, host_name):
+        logger.info(svc_messages.REMOVE_HOST.format(host_name))
```
Comment (ELENAGER) [inline controllers/array_action/array_mediator_svc.py:1799]: Delete host or remove host?

File: controllers/array_action/array_mediator_svc.py [hunk 2]
```diff
@@ -1870,6 +1884,7 @@ def _addhostiogrp(self, host_name, io_group):
     def add_io_group_to_host(self, host_name, io_group):
         if not io_group:
             return
+        logger.info(svc_messages.REMOVE_HOST_IO_GROUP.format(io_group, host_name))
```
Comment (ELENAGER) [inline controllers/array_action/array_mediator_svc.py:1887]: Maybe ADD_HOST_IO_GROUP?

File: controllers/array_action/array_mediator_svc.py (hunk 1 above)
Comment (matancarmeli7) [reply controllers/array_action/array_mediator_svc.py]: done

File: controllers/array_action/array_mediator_svc.py (hunk 2 above)
Comment (matancarmeli7) [reply controllers/array_action/array_mediator_svc.py]: done, nice catch

**Review by ELENAGER — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**